lambdanotes/src/components/CreateNote.css

```diff
@@ -0,0 +1,43 @@
+.createNote {
```

Comment:
I love how you are doing separation of concerns in your css. I like doing that too - the only trap I fall into is accidentally naming something the same thing. 

Maybe as a stretch try stylized components? Not sure how that would work though with redux. I've only ever used it with react.